english/staypuft/ghostbusters/protons/nothing-at-first/dejavu.md

```diff
@@ -0,0 +1,9 @@
+But wait, there in the distance... something is appearing.  Ever so slowly images are coming into view. The images appear closer and closer; faint and colorless at first, but more defined at each passing second. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.